README.md

```diff
@@ -1368,8 +1370,44 @@ handlers:
       - request_fields
 ```
 
+### Case 10. Add stack traces to log records
+
+If you want to embed the stack trace of either an Exception or a log entry in general, you can do so with following additions to the logging call:
+
+```python
+import sys
+
+logger.debug('My log with stack info', stack_info=True)
+logger.critical('Exception happened', exc_info=sys.exc_info())
+```
```

Comment:
Or you simply call `logger.exception('Your message')` that automatically adds the exc_info, however this one is logging as ERROR level and not CRITICAL.